HLTrigger/special/plugins/HLTPPSJetComparisonFilter.cc

```diff
@@ -83,7 +86,7 @@ void HLTPPSJetComparisonFilter::fillDescriptions(edm::ConfigurationDescriptions
   desc.add<bool>("do_xi", true)->setComment("flag to require xi matching");
   desc.add<bool>("do_my", false)->setComment("flag to require m,y matching");
 
-  descriptions.addWithDefaultLabel(desc);
+  descriptions.add("hltppsJetComparisonFilterDefault", desc);
```

Comment:
Actually, it's not the same. With `.add` I'm creating a file `hltppsJetComparisonFilterDefault_cfi.py`, and addWithDefaultLabel would create a file `hltppsJetComparisonFilter_cfi.py`. This is needed so that `hltppsJetComparisonFilter_cfi.py` I added in this could override and import params from `hltppsJetComparisonFilterDefault_cfi.py`.

Reply:
Reverted cause overriding `hltppsJetComparisonFilter_cfi.py` turned out to be unnecessary